net-p2p/mldonkey/mldonkey-3.1.7-r3.ebuild

```diff
@@ -141,9 +139,12 @@ src_install() {
 		done
 		use bittorrent && newbin "make_torrent${myext}" make_torrent
 
-		newconfd "${FILESDIR}/mldonkey.confd" mldonkey
-		fperms 600 /etc/conf.d/mldonkey
-		newinitd "${FILESDIR}/mldonkey.initd" mldonkey
+		if use systemd; then
+			systemd_dounit "${FILESDIR}/${PN}.service"
+		else
+			newconfd "${FILESDIR}/mldonkey.confd" mldonkey
+			newinitd "${FILESDIR}/mldonkey.initd" mldonkey
+		fi
```

Comment:
We always install the service files regardless of init system that's being used.

Reply:
Oh, OK, I didn't know that. So the ebuild should install both OpenRC and systemd service files unconditionally?
In that case I don't need `systemd` in `IUSE` because it's only used here.

Reply:
Fixed.